### PR TITLE
PLAN-176-made-game-view-fit-125%-screen-zoom

### PR DESCRIPTION
**Pull request by AndzejIlj**

### CODE REVIEW

File: services/app/src/components/chessBoard/chess-board.css [hunk 1]
```diff
@@ -57,115 +26,83 @@ table {
 }
 
 #chess-board .border-tile:hover, .non-border-tile:hover {
-  border: 1px solid #6547FD;
-  width: 70px;
-  height: 70px;
+  border: 1px solid var(--game-light-purple);
+  inset: 0;
```
Comment (jvalatkevicius) [inline services/app/src/components/chessBoard/chess-board.css:30]: why is this needed?

**Review by jvalatkevicius — CHANGES_REQUESTED** [review]

File: services/app/src/components/chessBoard/chess-board.css [hunk 2]
```diff
@@ -57,115 +26,83 @@ table {
...
+#chess-board .square .figure-img {
+  height: clamp(16px, 2vh,23px);
 }
 
 #chess-board .square .figure-text {
-  font-family: 'Rubik';
-  font-style: normal;
-  font-weight: 800;
-  font-size: 15px;
+  font-size: clamp(14px, 2vh, 16px);
```
[the remark this answers is not in the document]
Comment (AndzejIlj) [reply services/app/src/components/chessBoard/chess-board.css:96]: If you're talking about getting rid of all the clamps, I've deleted some of the them, but I think its a smart move to leave some as well.

File: services/app/src/components/chessBoard/chess-board.css [hunk 3]
```diff
@@ -57,116 +25,79 @@ table {
...
-  height: 23px;
+.square-move-info .square-move-text {
+  height: 25%;
+  z-index: 2;
 }
 
-#chess-board .square .figure-img-nth {
-  margin-left: -4px;
+#chess-board .square .figure-img {
+  height: clamp(16px, 2vh,23px);
```
Comment (jvalatkevicius) [inline services/app/src/components/chessBoard/chess-board.css:89]: isn't parent sqaure `clamp` enough?

**Review by jvalatkevicius — CHANGES_REQUESTED** [review]

File: services/app/src/components/chessBoard/chess-board.css (hunk 3 above)
Comment (AndzejIlj) [reply services/app/src/components/chessBoard/chess-board.css:89]: Its possible to do a constant height for it, but then it looks kind of inconsistent. If you think about setting height in percentages, then it does not work as intended.

File: services/app/src/components/gameFooter/chess-board-pieces.css [hunk 4]
```diff
@@ -21,15 +21,15 @@
...
+  border-color: var(--game-light-purple);
 }
 
 .piece-field.disabled {
   cursor: not-allowed;
 }
 
 .figure-img {
-  height: 35px;
+  height: clamp(30px , 3.5vh, 35px);
```
[the remark this answers is not in the document]
Comment (AndzejIlj) [reply services/app/src/components/gameFooter/chess-board-pieces.css:32]: Yes, it is, but we still need to set height for `figure-img`, because svgs can overflow its parent. And we need a height to make those svgs look consistent.

File: services/app/src/components/chessBoard/chess-board.css (hunk 3 above)
Comment (AndzejIlj) [reply services/app/src/components/chessBoard/chess-board.css:89]: Changed the height to 20px

File: services/app/src/components/gameFooter/chess-board-pieces.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (jvalatkevicius) [reply services/app/src/components/gameFooter/chess-board-pieces.css:32]: You should be able to make it fit into parent container.

**Review by jvalatkevicius — CHANGES_REQUESTED** [review]

File: services/app/src/components/chessBoard/Square.jsx [hunk 5]
```diff
@@ -37,45 +41,74 @@ const Square = ({
...
+          {turnToShow && (
+            <div>
+              <img 
+                src={figures[turnToShow.figure].img} 
+                alt={`${turnToShow.figure} icon`} 
+                className='figure-img'
+              />
+            </div>
+          )}
+          {turnToShow && (
```
Comment (jvalatkevicius) [inline services/app/src/components/chessBoard/Square.jsx:95]: these two have the same condition

**Review by jvalatkevicius — CHANGES_REQUESTED** [review]

**Review by jvalatkevicius — APPROVED** [review]
